### DISCUSSION TITLE
HEAD no longer resolves to the main branch

### DISCUSSION THREAD

**Discussion by SMillerDev** [community/community#15673] [Other Feature Feedback, Questions, & Ideas]
Homebrew uses the HEAD alias in the install URL, but that appears to have stopped working. Was this intentional? 

E.g. https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh was returning the same as https://raw.githubusercontent.com/Homebrew/install/master/install.sh until recently.

**Comment by dumbasPL** [comment]
This also breaks github.dev. If you try to open it from the main branch it tries to resolve HEAD and closes after getting a 404

**Comment by pablonete** [comment]
I found the culprit (look no further) and I'm working on a fix right now.

**Reply by pablonete** [reply]
Other potentially affected URLs with other symrefs like `HEAD@{5}`, e.g. https://github.com/nodejs/node/tree/HEAD%40%7B5%7D/src

**Reply by pablonete** [reply]
Issue to follow up and to add tests to prevent it happening in the future: https://github.com/github/repos/issues/1143

**Reply by pablonete** [reply]
Fix is now deployed to dotcom. 🎉 
Tested successfully with reported URLs.

**Reply by lishid** [reply]
Thank you for the speedy recovery!
